stub/stub.go

```diff
@@ -60,6 +60,7 @@ type Input struct {
 }
 
 type Output struct {
+	Delay int                    `json:"delay"`
```

Comment:
Just curious, do we need to just give time.Duration instead ? so it can give more flexibility for others to make the delay

```suggestion
	Delay time.Duration                    `json:"delay"`
```

or if not, just giving the comment on this field about the delay is in millisecond 
```suggestion
	Delay int                    `json:"delay"` // in milliseconds
```

Reply:
sure, Duration will be more flexible, then sticking to milliseconds. Updating pr.